src/Pages/Shared/ConfigMapSecret/utils.ts

```diff
@@ -265,7 +266,12 @@ export const getConfigMapSecretFormInitialValues = ({
             selectedType: type ?? configMapSecretMountDataMap.environment.value,
```

Comment:
Can we create a base struct that is common in both returns?

Reply:
We can destructure configMapSecretData above the if statement and re-apply same conditions like externalType, filePermission, roleARN, isResolvedData, yamlMode, isFilePermissionChecked and even more